src/models/Article.php

```diff
@@ -96,7 +101,9 @@ public function rules()
     public function attributeLabels()
     {
         return [
+            'id' => Module::t('id'),
             'title' => Module::t('article_title'),
+            'alias' => Module::t('article_alias'),
```

Comment:
the article_alias is missing in the translation file.

Reply:
I've added a translation for English, Russian and Ukrainian.